recipes-kernel/linux/files/freedom-u540/defconfig

```diff
@@ -36,7 +36,6 @@ CONFIG_PCIE_MICROSEMI=y
 CONFIG_PCI_SW_SWITCHTEC=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-CONFIG_EXTRA_FIRMWARE="radeon/BTC_rlc.bin radeon/CAICOS_mc.bin radeon/CAICOS_me.bin radeon/CAICOS_pfp.bin radeon/CAICOS_smc.bin radeon/SUMO_uvd.bin"
```

Comment:
Let me just test that this doesn't cause any breakages

Reply:
Yeah, removing this line breaks the firmware loading

Reply:
Can we keep this line in?

Converting the others to modules makes sense and doesn't change anything